docs/external_contributions.md

```diff
@@ -60,9 +60,13 @@ given by mentors. The contributor's role includes:
     and review.
 
 Credits **MUST** be given to external contributors. A _CREDITS_ file **MUST** be
-present at the root directory of the GE code repository, including one line per
-contributor to the project. An example can be found
-[here](https://github.com/nodejs/node/blob/master/AUTHORS).
+present at the root directory of the GE code repository. The format of that file
+MAY use some of the following alternatives (it's up to GE team to chose which
+option they prefer):
+
+- One line per contributor to the project. An example can be found [here](https://github.com/nodejs/node/blob/master/AUTHORS).
+- Include a link to the contributors graph in the github repository. An example of the contributor graph can be found [here](https://github.com/telefonicaid/fiware-orion/graphs/contributors).
```

Comment:
How about something like:

```
- A short list of major contributors and  a link to the contributors graph 
  in the github repository. An example can be found [here](https://github.com/Kotti/Kotti/blob/master/AUTHORS.txt)
```

Reply:
Sound good to me. It would be easy to mantation (major contributor are not expected to change often) and provide the link with the full information.

It opens a new question: where is the threeshold that separate regular contributors from major contributors. It is hard to define a criteria... maybe the best option is that GE team decide that.

Reply:
> the best option is that GE team decide that.

I agree the decision should be made by the regular GE team.